source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -0,0 +1,138 @@
+    - If that installation is still over the limit 7, 14, 30, and 90 days into the next billing cycle, they will be sent emails with various wording to get them to add a payment method and upgrade their cloud billing 
+    - If that installation is no longer over the limit 7, 14, and 30 days into the next billing cycle, we will "forgive" them, and not send any emails for that cycle. 
+    - After the installation has gone over the limit, each time the job runs, it checks Stripe to check the state of the subscription. If it's flipped from "past_due" to active, it will cancel any emails for that cycle. Otherwise, it will continue on.
+
+#### User Interface
+ - @-autocomplete results are now prioritized based on recency and thread activity.
+ - File attachments below size of 10 now show fractions.
+ - The formatting of the channel header change message was improved.
+ - Team invite workflow now shows BOT tags when the search returns Bot users.
+ - Added the ability to zoom in/out of PDF files.
+ - Added support for 16x16 base64 encoded mini images to use with progressive rendering.
+
+#### Notifications
+ - Channel-wide mentions are now automatically disabled when a user mutes a channel.
+
+#### Command Line Interface (CLI)
+ - Added new local API endpoints for getting, updating and deleting incoming and outgoing webhooks.
+ - Added mmctl system version endpoint to print the remote server version.
+ - Improved the mmctl system status command output to include all the reported values.
+
+#### Integrations
+ - Updated icon_emoji field in incoming webhooks to allow for emojis to also be specified with surrounding colons.
+ - Dynamic auto-completion is now supported for built-in slash commands.
+ - Added plugin hooks for ReationHasBeenAdded and ReactionHasBeenRemoved.
+
+#### Administration
+ - Added the ability to load a set of configuration custom defaults from a MM_CUSTOM_DEFAULTS_PATH environment variable.
+ - Added AWS metering service support.
+ - Added the ability to retrieve compliance files from the System Console.
+
+### Bug Fixes
+ - Fixed an issue with broken link previews for twitter links.
+ - Fixed an issue where editing a post did not submit with CMD+ENTER.
+ - Fixed an issue where users were able to create or edit slash commands to have more than two slashes in the URL.
+ - Fixed an issue where resized emojis got overwritten with original data.
+ - Fixed an issue where the sidebar category "More" menu was not shown when hovering over a long category name.
+ - Fixed an issue where a received direct message did not show up on the sidebar if it wasn't previously created.
+ - Fixed an issue where a search using from: failed to auto-load more results in the right-hand side when Elasticsearch was enabled.
+ - Fixed an issue where s3 file backend TestFileConnection failed due to permissions if S3PathPrefix was in use.
+ - Fixed an issue where an id was missing for Tooltip in PostInfo.
+
+### config.json
```

Comment:
We probably do not need this for external-facing docs since users cannot access this functionality.